### DISCUSSION TITLE
argocd backend plugin help

### DISCUSSION THREAD

**Discussion by supersol** [RoadieHQ/roadie-backstage-plugins#1402] [Q&A]
would you please update the documentation or provide some details how to use the argocd backend plugin with the new backstage backend system?

**Comment by wojalin** [comment]
A general question here: Why would I  like to use the `argocd backend plugin` at all? 
it seems to me, that it's for multiple ArgoCD instances (as option2) and you can interact with the ArgoCD resources.

Am I correct?

**Comment by juu0000** [comment]
i wanted too to know that argo backend plugin for new backend system.

i want to use multiple Argocd instance in a backstage.

and I'm curious. Is it possible to display the contents of two Argo instances resources in one component of backstage?
Listbox or any other form, anything is fine. I wish it will possible..
